pkg/downstream/suggest.go

```diff
@@ -64,28 +67,26 @@ func Suggest(ctx context.Context, git utils.GitHelper, client *github.Client, re
 			if len(msgLines) == 0 {
 				return fmt.Errorf("found commit with empty body: %s", c.GetSHA())
 			}
-			out, err := git.DoOutput("log", "--fixed-strings", "--grep", msgLines[0])
+			out, err := git.DoOutput("log", "--fixed-strings", "--pretty=format:%h", "--grep", msgLines[0])
```

Comment:
Why changing the formatting? Just asking as I don't have the context

Reply:
With `--pretty=format:%h` we make sure to print just the hash of each commit (found with `--grep`)
So that we can split `out` and get a slice of commit hashes